packages/text-field/theme/lumo/vaadin-input-field-shared-styles.js

```diff
@@ -53,15 +53,6 @@ registerStyles(
       transition: color 0.2s;
     }
 
-    [part='input-field'] ::slotted(:is(input, textarea)) {
-      cursor: inherit;
-      min-height: var(--lumo-text-field-size);
-      padding: 0 0.25em;
-      --_lumo-text-field-overflow-mask-image: linear-gradient(to left, transparent, #000 1.25em);
-      -webkit-mask-image: var(--_lumo-text-field-overflow-mask-image);
-      mask-image: var(--_lumo-text-field-overflow-mask-image);
-    }
-
```

Comment:
This CSS is duplicated here: https://github.com/vaadin/web-components/blob/91bfb8acc231ec7747bbebbfe2add5b83e647e00/packages/input-container/theme/lumo/vaadin-input-container-styles.js#L40-L46